magento/pwa-studio
# Increase peregrine/lib/apollo test coverage

## Description

Increase test coverage for the functions in `peregrine/lib/Apollo`

## Related Issue

Closes PWA-993

## Acceptance 

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps

1. Run: `yarn test --collectCoverageFrom=**/lib/Apollo/**/*.js packages/peregrine/lib/Apollo`
1. Verify tests succeed
1. Verify acceptable test coverage

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
